PFOffer.sol

```diff
@@ -98,6 +118,7 @@ contract PFOffer {
         oneTimeCosts = _oneTimeCosts;
         minDailyWithdrawLimit = _minDailyWithdrawLimit;
         dailyWithdrawLimit = _minDailyWithdrawLimit;
+        proposalID = _proposalID;
```

Comment:
Yes I had seen your implementation, but as you also point out it can be tricked by submitting two different proposals.

At the end I think that all these will be so easily detectable by the token holders that it does not matter at all. If such a thing happens the DAO observation team will put out an alert.

I am gonna port the function as is :)